examples/dmd/heat_conduction_csv.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+#SBATCH -N 1
+#SBATCH -t 1:00:00
+#SBATCH -p pbatch
+#SBATCH -o sbatch.log
+#SBATCH --open-mode truncate
+
+case "$(uname -s)" in
+    Linux*)
+		  COMMAND="srun -p pdebug"
+			MACHINE="Linux";;
+    Darwin*)
+		  COMMAND="mpirun -oversubscribe"
+			MACHINE="Darwin";;
+    *)
+			echo "The regression tests can only run on Linux and MAC."
+			exit 1
+esac
+
+rm -rf parameters.txt hc_list hc_data run/hc_data
+
+$COMMAND -n 1 ./parametric_heat_conduction -r 0.40 -offline -rdim 16 -csv -out hc_data/hc_par1 > hc_par1.log
```

Comment:
This script outputs many files to `build/examples/dmd`. It would be more convenient to write them to a subdirectory, e.g. `./out`, so that they can be removed easily.

Reply:
The command line is changed to prevent unnecessary outputs.